### ISSUE TITLE
Please get rid of SRCREV = "${AUTOREV}"

### ISSUE THREAD

**Issue by angelobt** [meta-flutter/meta-flutter#327]
It's impossible to have a reproducible setup.

**Comment by jwinarske** [comment]
@ardera

**Comment by jwinarske** [comment]
@angelobt Yes I completely agree.  Only ivi-homescreen and flutter-auto are locked to a commit.

To lock for either flutter-pi or Sony you simply use a bbappend file to override the SRCREV.

**Comment by ardera** [comment]
@jwinarske should we have some kind of auto-roller for flutter-pi then?

**Comment by jwinarske** [comment]
@ardera The common pattern is to only move the commit when there is a logical release, but that is 100% relative to the upstream repo and it's release patterns.

I'm not against a nightly CI job to auto-roll deps.  Anything to cut down on the maintenance overhead of this layer, I'm all for it.

**Comment by jwinarske** [comment]
https://github.com/meta-flutter/meta-flutter/pull/377
